pyproject.toml

```diff
@@ -15,7 +15,7 @@ enable = true
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
 
-airbyte-cdk = "^0.73.0"
+airbyte-cdk = "^0.81.6"
```

Comment:
This needs to be updated to match the version used in snowflake-cortex destination, ow the older version of langchain gets used, which creates a bunch of library conflicts.